CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/TokensFile.swift

```diff
@@ -0,0 +1,96 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SyntaxSupport
+import Utils
+
+let tokensFile = SourceFile(
+  leadingTrivia: .docLineComment(generateCopyrightHeader(for: "generate-swiftsyntax"))) {
+    FunctionDecl("""
+    fileprivate func defaultTrivia(presence: SourcePresence, trivia: Trivia) -> Trivia {
```

Comment:
Not really related to your PR but the `defaultTrivia` function appears to be unused, so you can just delete it.

Reply:
Will add it in a follow up PR